app/presenters/templates/components/form.latte

```diff
@@ -4,9 +4,17 @@
 </ul>
 
 <table>
-<tr n:foreach="$form->controls as $input" n:class="$input->required ? required">
-	<th>{label $input /}</th>
-	<td>{input $input} <span class=error n:ifcontent>{$input->error}</span></td>
-</tr>
+{foreach $form->controls as $input}
+    <tr n:if="!$input instanceof Nette\Forms\Controls\HiddenField" n:class="$input->required ? required">
```

Comment:
I'm personally not in favor of using getOption() for this and loved the instanceof approach - the options (and "type" in particular) can be used in many projects for many things. Or was that a bad practice? :)

Anyhow, updated the code to reflect